examples/tests/bootstrap/test-bootstrap.sh

```diff
@@ -136,4 +136,23 @@ if [[ $(stat --format '%G' "$HOME/.earthly/config.yml") != "$GRP" ]]; then
   echo "earthly config is not owned by the users group"
   stat "$HOME/.earthly/config.yml"
   exit 1
-fi
\ No newline at end of file
+fi
+
+echo "=== Test 6: Homebrew Source ==="
+
+bash=$("$earthly" bootstrap --source bash)
```

Comment:
What about expanding this test to ensure it still works when docker isn't present? e.g.

    DOCKER_HOST="bootstrap doesn't have docker" earthly bootstrap --source bash

Reply:
That should be a separate test with the `--no-buildkit` flag, as thats what would fail if docker is not present.

Reply:
my worry is that https://github.com/Homebrew/homebrew-core/blob/master/Formula/earthly.rb#L33-L36 will fail if docker isn't available, which would prevent the brew package from being built.

Reply:
I can add a test around that flag I mentioned earlier. We've already done releases with the buildkit docker stuff in there?

Reply:
I don't think we have:

```
alex@mah:~/gh/earthly/earthly$ DOCKER_HOST="bootstrap doesn't have docker" earthly-v0.5.13 bootstrap --source bash
complete -o nospace -C '/home/alex/bin/earthly-v0.5.13' earthly
alex@mah:~/gh/earthly/earthly$ DOCKER_HOST="bootstrap doesn't have docker" earthly bootstrap --source bash
           buildkitd | Is docker installed and running? Are you part of the docker group?
Error: bootstrap new buildkitd client: docker not available
```

Reply:
Its been in there since April 5 - PR #910. We released successfully 5x since then?

Reply:
I'll add a test for the `--no-buildkit` flag and add a note to update the recipe on next release.

Reply:
> add a note to update the recipe on next release

Are you talking about the homebrew forumula?

If yes, I don't understand why we can't maintain backwards compatibility and have earthly continue to work the way homebrew expects it to.

Reply:
Reolved. Added `DOCKER_HOST ` test (and others) for `--source`, and an individual test for `--no-buildkit`. We will break it out later to avoid this kind of confusion again.